modin/core/io/text/text_file_dispatcher.py

```diff
@@ -642,6 +642,11 @@ def check_parameters_support(
                 return False
         elif not cls.pathlib_or_pypath(filepath_or_buffer):
             return False
+        elif not (
+            hasattr(filepath_or_buffer, "name")
+            and cls.file_exists(filepath_or_buffer.name)
+        ):
```

Comment:
What does this do?

Reply:
@RehanSD changed the code a bit and added a comment. Ready for review